documentation/GETTING_STARTED.md

```diff
@@ -134,7 +134,16 @@ This can be done with the following command:
 ```shell
 sed -i'' -e 's/branch: \&branch main/branch: \&branch <FORK_BRANCH>/g' fiware-platform/values.yaml
 ```
-### 8. Apply the applications to the cluster
+
+### 8. Set the target namespace as the name of the release in the values.yaml
+
+Since all of your ArgoCD applications are going to live in the ArgoCD namespace, it is important to differentiate them with the name of the release, so a good strategy is to make it the name of the namespace you are deploying to:
+
```

Comment:
I think thats not required anymore. With the app-of-apps approach, you can set the destination namespace in the SOP-values.yaml of the platform. Using this command might even break this mechanism, since it will override default values.